### PR TITLE
Restaurant v0.1

### PR DESCRIPTION
**Pull request by Bismarck-GM**
# Milestone 1:

- Created the main HTML skeleton of the website using Bulma Framework.
- Initialized node and webpack.
- Add dependencies.
- Separate the JavaScript code in modules.
- Basically, index.js calls each module when each tab is clicked.
- Add data with objects as modules.

Thank you for the review 🌵

### CODE REVIEW

File: src/modules/template.js [hunk 1]
```diff
@@ -46,16 +46,16 @@ const PAGELOAD = (() => {
...
-      tabContent,
-      tabs
-  }
+    tabContent,
+    tabs,
+  };
 })();
 
 const TABS = PAGELOAD.tabs;
 const TABCONTENTS = PAGELOAD.tabContent;
```
Comment (lucasmazo23) [inline src/modules/template.js]: [OPTIONAL] Consider using object destructuring here, if you are trying to follow a certain naming convention, you can rename the variable in the destructuring statement:
```
const {tabs: TABS, tabContent: TABCONTENTS} = PAGELOAD
```

**Review by lucasmazo23 — CHANGES_REQUESTED** [review]
## Changes required 🔧
### You are doing a great job so far, and the project is looking good.

### For the project to get approved kindly implement these changes:

### Required changes:
- [x] [REQUIRED] Please rearrange your dist folder so that it follows the required distribution
  you should have a `main.js` in the dist folder and `index.html` (in the same folder) should be pointing to it, right now you have a `js/bundle.js`
image

image

[source](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page)

### Optional Changes
- [x] [OPTIONAL] There's a couple of places where you could be using object restructuring, and taking advantage from ES6 (check my comment below)
### Happy coding! 💻
## Lucas Mazo

**Comment by Bismarck-GM** [comment]
# Milestone 2:

- Edited `main.js` file path and creation method through webpack.
- Applied object destructuring as TSE pointed. Still not used to do these handy ES6 assignments.

Thanks for the review 🐴

File: src/modules/template.js (hunk 1 above)
Comment (Bismarck-GM) [reply src/modules/template.js]: Thank you Lucas for the review. Have a great weekend.

**Review by wathigo — APPROVED** [review]
### Approved :heavy_check_mark: 
Great job implementing the suggested changes.

**You may move on.**

**Simon Wathigo**

**Pull request merged**